travis/setup_conda_windows.sh

```diff
@@ -6,12 +6,14 @@ if [[ $DEBUG == True ]]; then
 fi
 
 if [[ -z "${MINICONDA_VERSION}" ]]; then
-    MINICONDA_VERSION=4.5.4
+    MINICONDA_VERSION=4.6.14
 fi
 
 echo "installing miniconda3"
 choco install miniconda3 --params="'/AddToPath:1'" --version="$MINICONDA_VERSION";
-export PATH="/c/tools/miniconda3/scripts:/c/tools/miniconda3/:$PATH";
+export PATH="/c/tools/miniconda/bin:$PATH"
```

Comment:
I think this needs to be `miniconda3`

Reply:
Oh yeah oops 😮

Reply:
I'm testing it in my own fork. Didn't seem to work the first time, so I added DEBUG and am still waiting.

Reply:
Does this need to be `/mnt/c`? https://superuser.com/questions/1113385/convert-windows-path-for-windows-ubuntu-bash

Reply:
hmm, ideally this should not be miniconda, but that we download (see my failed attempt here, still need to sort out some path issue: https://github.com/astropy/ci-helpers/compare/master...bsipocz:travis_windows_latest_miniconda)